app/views/application/_survey_widget.html.erb

```diff
@@ -0,0 +1,10 @@
+<% if current_user%>
+	<div class="alert alert-primary">Survey hidden because you are logged in.</div>
+<% elsif  Rails.application.config.hide_survey_sparrow %>
+	<div class="alert alert-primary">Survey hidden because hide_survey_sparrow is turned on.</div>
+<% else %>
+<div id="ss_survey_widget"></div>
+	<script id="SS_SCRIPT">
+		// moved this to the js file
+	</script>
```

Comment:
Does this empty <script> block need to be here then?

Reply:
The point of this file is basically just to put a `<div id="ss_survey_widget"></div>` target on the page? 

Do we really need/want those alerts in there? Or are they just temporary for debugging?  I don't think we want actual public users seeing the alert `Survey hidden because hide_survey_sparrow is turned on`... _ever_, right?

If this is going ot be just a single div, does it really need to be a separate partial?

Reply:
I wonder if there's a way to only put the div on the page for users who have not already taken the survey, so we don't need custom JS?

Reply:
I'll remove the console.log notices before we merge this. They do help at this stage in the process, though.

Reply:
Cool makes sense! To be clear, I have no problem with any console.log, even remaining in production. My concern is only with the on-screen HTML div alerts.

Reply:
Oh, yeah, those are definitely going away before we merge.

Reply:
Yes, the script block needs to be there, or else there will be an error in the canned JS we get from SurveySparrow. I could edit that JS to avoid the error, but we don't want to go any further down that road.